packages/server/src/service/trpc.ts

```diff
@@ -96,6 +97,15 @@ export class VbmRpc implements ImplRpc<IVbmRpc, Request> {
       return e
     }
   }
+  public isRegistered = async (voter: StateInfo) => {
```

Comment:
`Promise<RpcRet<boolean>>` means you need to return `data(x)` or `error(x)` (see the other calls).